packages/vendure-plugin-sales-per-variant/README.md

```diff
@@ -0,0 +1,125 @@
+# Vendure Sales Per Variant plugin
+
+![Vendure version](https://img.shields.io/npm/dependency-version/vendure-plugin-metrics/dev/@vendure/core)
+
+### [Official documentation here](https://pinelab-plugins.com/plugin/vendure-plugin-metrics)
+
+A plugin to measure and visualize your shop's conversion rate (CRV), average order value (AOV) and number of orders per
+month or per week, for the past 12 months (or weeks) per variants.
+
+## Getting started
+
+1. Configure the plugin in `vendure-config.ts`:
+
+```ts
+import { MetricsPlugin } from "vendure-plugin-sales-per-variant";
+
+plugins: [
+  ...
+    SalesPerVariantPlugin,
+    AdminUiPlugin.init({
+      port: 3002,
+      route: 'admin',
+      app: compileUiExtensions({
+        outputPath: path.join(__dirname, '__admin-ui'),
+        extensions: [SalesPerVariantPlugin.ui],
+      }),
+    }),
+  ...
+]
+```
+
+2. Start your Vendure server and login as administrator
+3. You should now be able to select `metrics` when you click on the button `add widget`
```

Comment:
let's rename this to `Product metrics` to avoid conflicts with the other metrics plugin